response_operations_ui/views/collection_exercise.py

```diff
@@ -1138,7 +1138,17 @@ def view_sample_ci_summary(short_name: str, period: str) -> str:
 @login_required
 def view_ci_versions(short_name: str, period: str, form_type: str) -> str:
 
-    return render_template("collection_exercise/ci-versions.html", form_type=form_type)
+    breadcrumbs = [
+        {"text": "Back to CIR versions", "url": "/surveys/" + short_name + "/" + period + "/view-sample-ci/summary"},
+        {},
+    ]
+    return render_template("collection_exercise/ci-versions.html", form_type=form_type, breadcrumbs=breadcrumbs)
+
+
+@collection_exercise_bp.route("/<short_name>/<period>/view-sample-ci/summary/<form_type>", methods=["POST"])
+@login_required
+def save_ci_versions(short_name: str, period: str, form_type: str) -> str:
+    return redirect(url_for("collection_exercise_bp.view_collection_exercise", short_name=short_name, period=period))
```

Comment:
Exactly what the card is asking for, but it feels wrong to me, I think it should be sending the user back to the 'previous' page, the step 2 of 2 with the big table since they might be going through the CIs one by one to update the versions. But that might get done on a different card

Reply:
As discussed, I won't be changing something that was refined by the whole team